src/components/templates/Search/index.tsx

```diff
@@ -33,14 +34,13 @@ export default function SearchPage({
 
   useEffect(() => {
     if (!appConfig.metadataCacheUri) return
+
+    const source = axios.CancelToken.source()
```

Comment:
We have to use the cancel token within this `useEffect`, in its`return` callback so that unmounting it really cancels the request